ncm-autofs/src/main/pan/components/autofs/schema.pan

```diff
@@ -67,6 +76,6 @@ type autofs_component = {
     "preserveMaster" : boolean = true
     @{This resource contains one entry per autofs map to manage. The dict key is
     mainly an internal name but it will be used to build the default map name.}
-    "maps" : autofs_map_type{}
+    "maps" ? autofs_map_type{}
     "conf" ? autofs_conf
 };
```

Comment:
this needs extra logic to fail when there is no `maps` and no (or none?) amd type autofs mounts